docs/deploy.md

```diff
@@ -135,3 +135,54 @@ frontend:
 | /<*>.md        | /<*>.md        | 200 (Rewrite) |
 | /<*>.png       | /<*>.png       | 200 (Rewrite) |
 | /<*>           | /index.html    | 200 (Rewrite) |        
+
+
+## Docker
+
+>There is a simple demo for building docsify in docker.
+
+?> tips: you can get all the files for this demo from [here](https://github.com/Koooooo-7/docsify-docker).
```

Comment:
The link to the separate repo containing sample files should be removed. These files are easily generated by `docsify-cli` or manually created as described in the [Quickstart](https://docsify-preview-csiiei9ph.vercel.app/#/quickstart?id=manual-initialization) section of the docs. We don't want another set of files to maintain.